vignettes/function-flow.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+title: "Function Flow"
+output: rmarkdown::html_vignette
+bibliography: references.bib
+csl: vancouver.csl
+vignette: >
+  %\VignetteIndexEntry{Function Flow}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Function conventions
+
+The below conventions are *ideals* only, to be used as a guidelines to
+help with development and understanding of the code. They are not hard
+rules.
+
+### Naming
+
+-   First word is an action verb, later words are objects or conditions.
+-   Exclusion criteria are prefixed with `exclude_`.
+-   Inclusion criteria are prefixed with `include_`.
+-   Helpers that get or extract a condition ("pregnancy" or "date of
+    visit") are prefixed with `get_`.
+-   Helpers that drop or keep a specific condition are prefixed with
+    `drop_` or `keep_` ("first visit date to maternal care for pregnancy
+    after 40 weeks"). These types of helpers likely are contained in the
+    `get_` functions.
+
+### Input and output
+
+-   Few arguments, with one or two core required argument.
+-   `include_` functions take a register as the first argument.
+    -   One input register database at a time.
+-   `exclude_` functions can take a register as the first argument or
+    take the output from an `include_` function.
+-   Second argument can be an output data from another function.
+
+## Function flow
+
+The OSDC algorithm - and thereby, the `osdc` package - contains one main
+function that will classify individuals into those with either type 1 or
+type 2 diabetes using the Danish registers: `classify_diabetes()`. This
```

Comment:
```suggestion
type 2 diabetes using the Danish registers: `classify_diabetes_status()`. This
```
And then output will include all individuals that were in the input data and not only individuals with diabetes, as we, @Aastedet and I, discussed at some point?